javascript/javascript2/week1/shortestWord/shortest-word.js

```diff
@@ -0,0 +1,15 @@
+"use strict"
+/**Write a function that finds the shortest word of an array of words */
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+
+function getShortestWord(danishWords) {
+    let shortestWord = danishWords[0];
+    for (const currentWord of danishWords) {
+        if (shortestWord.length > currentWord.length) {
```

Comment:
In the first loop iteration your code will always compare the first word with itself!
E.g., here in the first iteration both `currentWord` and `shortestWord` are "bil"
Which means there is always one unnecessary iteration. Can you change the for loop such that this is solved?

Reply:
Thank you for reviewing my homework. I have changed the loop